SocialCareCaseViewerApi.Tests/V1/IntegrationTests/MASH/UnAssignWorkerFromMashReferralTests.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using FluentAssertions;
+using NUnit.Framework;
+using SocialCareCaseViewerApi.Tests.V1.Helpers;
+using JsonSerializer = System.Text.Json.JsonSerializer;
+
+namespace SocialCareCaseViewerApi.Tests.V1.IntegrationTests.MASH
+{
+    [TestFixture]
+    public class UnAssignWorkerFromMashReferralTests : IntegrationTestSetup<Startup>
+    {
+        private SocialCareCaseViewerApi.V1.Infrastructure.MashReferral _existingDbReferral;
+
+        [SetUp]
+        public void Setup()
+        {
+            // Clear test database of any rows in the database
+            DatabaseContext.Teams.RemoveRange(DatabaseContext.Teams);
+            DatabaseContext.Workers.RemoveRange(DatabaseContext.Workers);
+            DatabaseContext.MashReferrals.RemoveRange(DatabaseContext.MashReferrals);
```

Comment:
The test database should be cleared before each test based on the setup method define the IntegrationTestSetup class. Do the tests fail without these lines? We might need to fix the initial setup if that's the case.

Reply:
Copy paste from existing test ( will refactor that too )

Reply:
Checked the IntegrationTestSetup it should wipe before and after, tested and it works indeed.